### PR TITLE
add hw file and update README.md

### PR DESCRIPTION
**Pull request by Davison322**
Update Laba5

### CODE REVIEW

File: Laba5.cpp [hunk 1]
```diff
@@ -0,0 +1,110 @@
...
+	TERRESTRIAL,
+	JOVIAN
+};
+
+class Planetary
+{
+	public:
+	vector<Planet> listOfPlanets;
+
+	public:
```
Comment (SolomiaSydoryk) [inline Laba5.cpp:40]: no need to specify the field type "public" again

File: Laba5.cpp [hunk 2]
```diff
@@ -0,0 +1,110 @@
...
+
+	void listOfPlanetsAdd(Planet somePlanet)
```
Comment (SolomiaSydoryk) [inline Laba5.cpp:42]: The name of the function is capitalized (ListOfPlanetsAdd)

File: Laba5.cpp [hunk 3]
```diff
@@ -0,0 +1,110 @@
...
+enum type
+{
+	TERRESTRIAL,
+	JOVIAN
+};
+
+class Planetary
+{
+	public:
+	vector<Planet> listOfPlanets;
```
Comment (SolomiaSydoryk) [inline Laba5.cpp:38]: change from "public" to "private"

File: Laba5.cpp [hunk 4]
```diff
@@ -0,0 +1,110 @@
...
+using namespace std;
+
+class Planet
+{
+	
+	friend class Planetary;
+
+	private:
+		string name;
+		int massInKg;
```
Comment (SolomiaSydoryk) [inline Laba5.cpp:12]: incorrect variable entry
should be written:
int mass_in_kg;

File: Laba5.cpp [hunk 5]
```diff
@@ -0,0 +1,110 @@
...
+		int orbitalVelocity;
+		double meanTemperature;
+		double lenghtOfDay;
```
Comment (SolomiaSydoryk) [inline Laba5.cpp:15]: incorrect variable entry
should be written:
int orbital_velocity;
double mean_temperature; or double value_of_temperature;
double lenght_of_day;

File: Laba5.cpp [hunk 6]
```diff
@@ -0,0 +1,110 @@
...
+		int distanceFromSun;
```
Comment (SolomiaSydoryk) [inline Laba5.cpp:16]: incorrect variable entry
should be written:
int distance_from_sun;

File: Laba5.cpp [hunk 7]
```diff
@@ -0,0 +1,110 @@
...
+	public:
+		Planet(string namee, int mass, int velocity, double temperature, double lenghtofday, int distance)
+		{
+			name = namee;
+			massInKg = mass;
+			orbitalVelocity = velocity;
+			meanTemperature = temperature;
+			lenghtOfDay = lenghtofday;
+			distanceFromSun = distance;
+		};
```
Comment (SolomiaSydoryk) [inline Laba5.cpp:26]: incorrect variable entry
should be written:
double lenght_of_day;

you can use "this->" for better recording
for example:
Planet(string name, int mass_in_kg, int orbital_velocity, double mean_temperature, double lenght_of_day, int distance_from_sun) {
this->name=name;
this->mass_in_kg=mass_in_kg; 
....
};

File: Laba5.cpp [hunk 8]
```diff
@@ -0,0 +1,110 @@
...
+	{
+		cout << firstPlanet.distanceFromSun + secondPlanet.distanceFromSun << endl;
+	}
+
+	void print()
+	{
+		for (int i = 0; i < listOfPlanets.size(); i++)
+		{
+			cout << listOfPlanets[i].name << " ";
+		}
```
Comment (SolomiaSydoryk) [inline Laba5.cpp:57]: The name of the function is capitalized

File: Laba5.cpp [hunk 9]
```diff
@@ -0,0 +1,110 @@
...
+			for (j = 0; j < n - i - 1; j++)
+			{
+				if (listOfPlanets[j].distanceFromSun > listOfPlanets[j + 1].distanceFromSun)
+				{
+					swap(listOfPlanets[j], listOfPlanets[j + 1]);
+				}
+			}
+		}
+
+	}
```
Comment (SolomiaSydoryk) [inline Laba5.cpp:76]: The name of the function is capitalized

File: Laba5.cpp [hunk 10]
```diff
@@ -0,0 +1,110 @@
...
+
+	static double findAverageMass(vector<Planet> listOfPlanets)
```
Comment (SolomiaSydoryk) [inline Laba5.cpp:78]: static double FindAverageMass(vector<Planet> ListOfPlanets)

File: README.md [hunk 11]
```diff
@@ -1 +1,11 @@
...
+
+Додати enum type: TERRESTRIAL, JOVIAN
+
+Створити клас Planetary,  котрий складається з певної кількості планет
+
+Додати функцію сортування за довжиною світлового дня
+
+Додати функцію findDistanceBetween(planetA, planetB)
+
+Додати функцію findAverageMass( Planet* )
```
Comment (SolomiaSydoryk) [inline README.md:11]: Назви функцій з великої літери

File: Laba5.cpp (hunk 3 above)
Comment (SolomiaSydoryk) [reply Laba5.cpp]: or delete the next "public" (on line 40), you already have it, why again?

File: Laba5Header.h [hunk 12]
```diff
@@ -0,0 +1,89 @@
...
+			name = namee;
+			mass_in_kg = mass;
+			orbital_velocity = velocity;
+			mean_temperature = temperature;
+			lenght_of_day = lenght_of_day;
+			distance_from_sun = distance;
+		};
+};
+
+enum type
```
Comment (SolomiaSydoryk) [inline Laba5Header.h:29]: Enumerators (for both scoped and unscoped enums) should be named like constants, not like macros. About all of them
For example

Знімок екрана_20230127_203035

File: Laba5.cpp [hunk 13]
```diff
@@ -0,0 +1,20 @@
+#include "Laba5Header.h"
+
+int main()
+{
+	Planet PlanetA("PlanetA", 1488, 77777, 14.5, 123456789, 500000);
```
Comment (zveres-ss) [inline Laba5.cpp:5]: variables should always start with lowerase

File: Laba5Header.h [hunk 14]
```diff
@@ -0,0 +1,89 @@
...
+{
+	TERRESTRIAL,
+	JOVIAN
+};
+
+class Planetary
+{
+	public:
+	vector<Planet> listOfPlanets;
```
Comment (zveres-ss) [inline Laba5Header.h:38]: never use two different conventions in your code together:
int orbital_velocity; this is in previous class
vector<Planet> listOfPlanets; - here

either caselCase or snake_case

**Review by zveres-ss — CHANGES_REQUESTED** [review]

**Pull request closed without merging**